packages/core/chrome/core-chrome-browser-internal/src/project_navigation/project_navigation_service.ts

```diff
@@ -222,15 +223,16 @@ export class ProjectNavigationService {
    * @param navTreeDefinition$ The navigation tree definition
    * @param location Optional location to use to detect the active node in the new navigation tree
    */
-  private initNavigation(
-    navTreeDefinition$: Observable<NavigationTreeDefinition>,
-    location?: Location
-  ) {
+  private initNavigation(id: string, navTreeDefinition$: Observable<NavigationTreeDefinition>) {
```

Comment:
nit: `@param` doesn't match functions params

Reply:
Done in https://github.com/elastic/kibana/pull/179270/commits/3d76ee313992cc58b6119cae00b53404284bf08c